frontend/src/components/chats/components/current-chat/current-chat.tsx

```diff
@@ -0,0 +1,46 @@
+import {
+  ChatMessageGetAllItemResponseDto,
+  ChatMessageUserResponseDto,
+  FC,
+} from 'common/types/types';
+
+import { MessageForm, MessagesList } from './components/components';
+import styles from './styles.module.scss';
+
+type Props = {
+  chatId: string | null;
+  chatOpponent: ChatMessageUserResponseDto | undefined;
+  currentUserId: number;
+  messages: ChatMessageGetAllItemResponseDto[];
+};
+
+const CurrentChat: FC<Props> = ({
+  chatId,
+  chatOpponent,
+  currentUserId,
+  messages,
+}) => {
+  return (
+    <div className={styles.currentChatWrapper}>
+      <div className={styles.currentChatHeader}>
+        {chatOpponent && <h4>{chatOpponent.userDetails.fullName}</h4>}
+      </div>
+      <div className={styles.currentChatContent}>
+        {messages.length ? (
```

Comment:
```suggestion
        {Boolean(messages.length) ? (
```

Reply:
Once @v0ldemar01 advised me to do the length check in the way I have done here and it was OK. I think here it is not confusing but coaches advice would be better

Reply:
prefer `hasMessages`